cmd/goQuery/goQuery.go

```diff
@@ -314,10 +315,12 @@ func throwMsg(msg string, external bool, fmtSpec string) {
 	if fmtSpec == "json" {
 		// If called non-interactively, write full error message to message log
 		if external {
-			if goDB.InitDBLog() != nil {
+			// TODO: change this to syslog
```

Comment:
Is this left undone on purpose (i.e. TBD later) or was it just forgotten during implementation testing?